backend/lambda/database_calls.py

```diff
@@ -35,18 +36,27 @@ def __init__(self):
         cloud_sql_connection_name = 'deepcite-306405:us-central1:deepcite'#os.environ["CLOUD_SQL_CONNECTION_NAME"]
 
         pool = sqlalchemy.create_engine(
-
+            # Equivalent URL:
+            # postgresql+pg8000://<db_user>:<db_pass>@<db_host>:<db_port>/<db_name>
             sqlalchemy.engine.url.URL(
                 drivername="postgresql+pg8000",
                 username=secret['username'],  # e.g. "my-database-user"
                 password=secret['password'],  # e.g. "my-database-password"
-                database=secret['dbInstanceIdentifier'],  # e.g. "my-database-name"
-                query={
-                    "unix_sock": "{}/{}/.s.PGSQL.5432".format(
-                        db_socket_dir,  # e.g. "/cloudsql"
-                        cloud_sql_connection_name)  # i.e "<PROJECT-NAME>:<INSTANCE-REGION>:<INSTANCE-NAME>"
-                }
+                host=secret['host'],  # e.g. "127.0.0.1"
+                port=secret['port'],  # e.g. 5432
+                database=secret['dbInstanceIdentifier']  # e.g. "my-database-name"
             ),
+            # sqlalchemy.engine.url.URL(
```

Comment:
Do we still want this commented code?

Reply:
Honestly, this function works locally but I'm not sure it will work on GCP. I was going to keep it in just in case but it would be much cleaner just to delete it and add it back in if I need it.